docs/sphinx/source/_static/bracketed_reference_number.css

```diff
@@ -0,0 +1,6 @@
+.footnote-reference::before {
+  content: '[';
+}
+.footnote-reference::after {
+  content: ']';
+}
```

Comment:
Could consolidate this with the other footnote CSS file since they're both targeting `footnote-reference`, nbd though since the formatting is unrelated

Reply:
yes, updated